ksql-parser/src/test/java/io/confluent/ksql/parser/KsqlParserTest.java

```diff
@@ -717,4 +811,79 @@ public void shouldSetShowDescriptionsForShowQueriesDescriptions() {
     ListQueries listQueries = (ListQueries)statement;
     Assert.assertThat(listQueries.getShowExtended(), is(true));
   }
+
+  @Test
+  public void shouldFailIfStreamColumnNameIsAmbiguous() {
+    final String statementString =
```

Comment:
Why is this ambiguous?

Reply:
This is based on the above assumption that if you have a column with the same name as stream name/alias you have to always prefix the stream name/alias to use that column.